sample-apps/react/messenger-clone/src/components/ChannelPreview/ChannelPreviewCallControls.tsx

```diff
@@ -0,0 +1,39 @@
+import {
+  AcceptCallButton,
+  CallingState,
+  CancelCallButton,
+  useCall,
+} from '@stream-io/video-react-sdk';
+import { useChatContext } from 'stream-chat-react';
+import { useCallCallingState } from '../../hooks/useCallCallingState';
+
+export const ChannelPreviewCallControls = () => {
+  const { channel: activeChannel } = useChatContext();
+  const call = useCall();
+  const callingState = useCallCallingState();
+
+  const callingToActiveChannel =
+    activeChannel && call && activeChannel.id === call.data?.custom.channelId;
```

Comment:
```suggestion
  const metadata = useCallMetadata();
  const callingToActiveChannel =
    activeChannel && call && activeChannel.id === metadata?.custom.channelId;
```

Reply:
We should probably use `cid` instead of `id`.

Reply:
changed all the usages of channel id to channel cid